src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepositorySpec.scala

```diff
@@ -85,18 +108,61 @@ class FileMetadataRepositorySpec extends AnyFlatSpec with ScalaFutures with Matc
     val consignmentId = UUID.fromString("c6f78fef-704a-46a8-82c0-afa465199e66")
     val fileOneId = "20e0676a-f0a1-4051-9540-e7df1344ac11"
     val fileTwoId = "b5111f11-4dca-4f92-8239-505da567b9d0"
-    val propertyId = "7a1b272c-e2f7-4b8f-8291-5e9dc312edb7"
     val metadataId = "f4440f43-20c6-4b6c-811d-349e633617e5"
 
     TestUtils.createConsignment(consignmentId, userId)
     TestUtils.createFile(UUID.fromString(fileOneId), consignmentId)
     TestUtils.createFile(UUID.fromString(fileTwoId), consignmentId)
+    TestUtils.addFileProperty(SHA256ServerSideChecksum)
 
-    (1 to 7).foreach { _ => TestUtils.addFileMetadata(UUID.randomUUID().toString, fileOneId, propertyId)}
+    (1 to 7).foreach { _ => TestUtils.addFileMetadata(UUID.randomUUID().toString, fileOneId, SHA256ServerSideChecksum)}
 
-    TestUtils.addFileMetadata(metadataId, fileTwoId, propertyId)
+    TestUtils.addFileMetadata(metadataId, fileTwoId, SHA256ServerSideChecksum)
     val fileMetadataFiles = fileMetadataRepository.countProcessedChecksumInConsignment(consignmentId).futureValue
 
     fileMetadataFiles shouldBe 2
   }
+
+  "addFileMetadata" should "add metadata with the correct values" in {
```

Comment:
Maybe check data has been added to the DB as part of this test?

Reply:
I've added in a method to check the metadata exists.